src/System.CommandLine.StaticCompletions/System.CommandLine.StaticCompletions.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(SdkTargetFramework)</TargetFramework>
```

Comment:
I think this would have to change if you wanted to make it a separate library

Reply:
Totally agree - this was one of my few compromises in this regard because I wasn't sure how long it might be before we could release this. We'd need a stable S.CL version before we could reasonably do that, and I don't know a timeline for that at the moment.

Reply:
Totally agree - this was one of my few compromises in this regard because I wasn't sure how long it might be before we could release this. We'd need a stable S.CL version before we could reasonably do that, and I don't know a timeline for that at the moment.